examples/providers/postgres/semantic-search.ipynb

```diff
@@ -0,0 +1,493 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using the Pinecone Retrieval App\n",
```

Comment:
Pinecone? maybe need a gsub /Pinecone/Postgres/, but it also looks like there is some other pinecone specific stuff in here.